File: SE1JAVA/JavaExam1Homewark/3-kth-largest-element-java/src/main/java/KthLargestElement.java

public class KthLargestElement {

	public static int findKthLargest(int[] nums, int k) {
		for (int i = 0; i < nums.length - 1; i++) {
			for (int i1 = 0; i1 < nums.length - 1 - i ; i1++) {
				if (nums[i1] < nums[i1 + 1]){
					int tmp = nums[i1];
					nums[i1] = nums[i1 + 1];
					nums[i1 + 1] = tmp;
				}
			}
		}
		return nums[k - 1];
	}
}

File: SE1JAVA/JavaExam1Homewark/9-kth-largest-element-cplusplus/src/kth-largest-element.cc
#include "kth-largest-element.h"

#include <iostream>
using namespace std;





int findKthLargest(int a[], int n, int K) {
    for (int i = 0; i < n - 1; i++) {
			for (int i1 = 0; i1 < n - 1 - i ; i1++) {
				if (a[i1] < a[i1 + 1]){
					int tmp =   a[i1];
					a[i1] = a[i1 + 1];
					a[i1 + 1] = tmp;
				}
			}
		}
		return  a[K - 1];

}



